doc/conf.py

```diff
@@ -95,6 +97,18 @@ def _check_deps():
   'pandas': ('http://pandas.pydata.org/pandas-docs/stable', None)
   }
 
+explicit_order_folders = ['../examples/pyplots',
+                          '../examples/statistics',
+                          '../examples/color',
+                          '../examples/pylab_examples',
+                          '../examples/showcase',
+                          '../tutorials/introductory',
```

Comment:
The way SG handles this is there's just one list that defines the order priority. So folders inside of tutorials and examples won't be shown together or anything, this just defines the ordering. You can think of this as two separate lists, one for examples and one for tutorials.